addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -303,7 +303,7 @@ div.media_iframe_video {
 
     iframe {
         width: 100%;
-        height: 100%;
+        height: 80%;
```

Comment:
Doesn't look right solution to me, why don't you assign ```height: 100vh;``` to it's parent element?

Reply:
I'll try this solution and get back to you afterward.

Reply:
Hello @msh-odoo, 

I have made changes and improve css of `div.media_iframe_video` and `iframe`.